vortex-array/src/arrays/varbinview/mod.rs

```diff
@@ -382,6 +383,12 @@ impl VarBinViewArray {
         &self.buffers
     }
 
+    /// Share ownership of the underlying raw data buffers, not including the views buffer.
+    #[inline]
+    pub(crate) fn buffers_arc(&self) -> Arc<[ByteBuffer]> {
```

Comment:
If you made `buffers(&self)` return `&Arc<[ByteBuffer]>` wouldn't that almost always work as intended? Since it could deref to `&[ByteBuffer]`

Reply:
yea thought about that but found it a bit weird to return a &Arc, but yea that should cover both cases